Sources/LocalizationCore/Localization.swift

```diff
@@ -46,21 +46,43 @@ public final class Localization {
             }
             
             try values.forEach { key, values in
-                let output = values.map { $0.localizableRow }.joined(separator: "\n")
-                let dir = NSString(string: outputDir).expandingTildeInPath + "/" + key + ".lproj"
-                let file = dir + "/" + fileName
-                
-                try? FileManager.default.removeItem(atPath: file)
-                try? FileManager.default.createDirectory(atPath: dir, withIntermediateDirectories: true, attributes: nil)
-                
-                do {
-                    try output.write(toFile: file, atomically: true, encoding: .utf8)
-                } catch {
-                    throw LocalizationError.unableToWrite
+                let outputArray = values.map { $0.localizableRow }
+                let output = outputArray.filter { !$0.contains("plist.") }.joined(separator: "\n")
```

Comment:
I'd move the `plist.` string somewhere to a constant so it can be easily changed in the future, maybe if #3 would be eventually implemented it might be a good candidate to be set using some argument

Reply:
Added it to `Constants.swift`.